lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -6,16 +6,16 @@ import {
   SubmissionRequirement,
 } from '@sphereon/pe-models';
 
-import { Checked, Status } from '../ConstraintUtils';
-import { JsonPathUtils } from '../utils';
-import { VerifiableCredential, VerifiablePresentation, VP } from '../verifiablePresentation';
-import { Presentation } from '../verifiablePresentation/models';
+import {Checked, Status} from '../ConstraintUtils';
+import {JsonPathUtils} from '../utils';
+import {VerifiableCredential, VerifiablePresentation, VP} from '../verifiablePresentation';
+import {Presentation} from '../verifiablePresentation';
 
-import { SelectResults } from './core/selectResults';
-import { SubmissionRequirementMatch } from './core/submissionRequirementMatch';
-import { EvaluationClient } from './evaluationClient';
-import { EvaluationResults } from './evaluationResults';
-import { HandlerCheckResult } from './handlerCheckResult';
+import {SelectResults} from './core';
+import {SubmissionRequirementMatch} from './core';
+import {EvaluationClient} from './evaluationClient';
+import {EvaluationResults} from './evaluationResults';
+import {HandlerCheckResult} from './handlerCheckResult';
```

Comment:
I am not sure what changed. But it it is showing erros from the build folders. 
Can @ksadjad @maikel-maas share more info ?

Reply:
Actually we saw this before, Intellij applies its own style. It happens with spacing and methods that becoming static.

Reply:
I don't understand what you mean by error in build folders but I do know that you should have space before and after your import items here. and the build is failing because of that

Reply:
fixed